pkg/operator/controller/ingress/deployment.go

```diff
@@ -488,6 +514,41 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 	return deployment, nil
 }
 
+// accessLoggingForIngressController returns an AccessLogging value for the
```

Comment:
@Miciah accessLoggingForIngressController returns a pointer not a value, right?

Reply:
Right, it's a pointer.  We can fix the comment in a follow-up.